src/database/PersonCategories/index.ts

```diff
@@ -0,0 +1,58 @@
+import dbConnection, { tableNames, execute } from '../connection';
+import logger from '@/utilities/modules/logger';
+
+const personCategoriesTable = () => dbConnection()(tableNames.PERSON_CATEGORIES);
+const personCategoryAttributesTable = () => dbConnection()(tableNames.PERSON_CATEGORY_ATTRIBUTES);
+
+interface PersonCategory {
+  personcategoryid: number;
```

Comment:
I think these should be camel case

Reply:
Surprise surprise, they can't be in postgres. Everything is automatically made lowercase in postgres, so to make a struct to parse them, all fields have to be completely lowercase :) 

Seriously, take a quick read through this. 
http://blog.lerner.co.il/quoting-postgresql/

Or the postgres stuff could be really confusing.